PP_APP/server/pp_api/api/urls.py

```diff
@@ -0,0 +1,20 @@
+from django.urls import path
+from pp_api.api.views import (
+    get_hello_rest,
+    update_hello_rest,
+    delete_hello_rest,
+    create_hello_rest,
+    upload_imgs,
+    display_feed_view,
+)
+
+app_name = 'pp_api'
+
+urlpatterns = [
+    path('display_feed', display_feed_view.as_view(), name='display'),
+    path('upload', upload_imgs, name='upload'),
+    path('get_hello', get_hello_rest, name='detail'),
+    path('update_hello', update_hello_rest, name='detail'),
+    path('delete_hello', delete_hello_rest, name='detail'),
+    path('create_hello', create_hello_rest, name='detail'),
```

Comment:
Just something to consider. https://www.youtube.com/watch?v=a48xeeo5Vnk&t=1070s at 17:50 this guy says to leave a trailing slash on the ending of the url paths because then it will treat "/" or no "/" as the same.

Reply:
I think its convention for REST APIs not to include trailing slash: https://blog.restcase.com/7-rules-for-rest-api-uri-design/